### DISCUSSION TITLE
阿里云盘的排序能否优化下呢

### DISCUSSION THREAD

**Discussion by qadan72** [AlistGo/alist#808] [Ideas]
就像这样，我在文件名手动加了01,02,03,04,05·······，排序还是有问题
![image](https://user-images.githubusercontent.com/40349902/159864489-e17b262f-5e47-4d21-811f-9a4eccd810ad.png)

**Comment by xhofe** [comment]
阿里云盘的排序使用的是他自己的api。
你这个图里的也不是01、02啊。
